features/user-management.feature

```diff
@@ -0,0 +1,142 @@
+Feature: User Management
+  There are two classes of users who can log into the system.
+  Most users will be ordinary, aka non-admin users.  Some
+  will have admin privileges.
+
+  Admin users have the ability to change any profiles and
+  delete any account but his own.
+
+  * A user who is not signed up cannot access any of the
+    user management buttons on the Users page.
+  * A  non-admin user cannot add a new user
+  * A non-admin user can manage his own profile information,
+    but cannot delete his account.
+
+  @javascript
+  Scenario: A user can sign into the system with registered email and password
+    Given I am not signed in
+    When I navigate to the Sign In page
+    And I enter a registered email
+    And I enter the associated password
+    And I click Sign In
+    Then I am sent to the Profile page
+    And I see the new user name
+    And I see a success message containing "Signed in successfully"
+
+  @javascript
+  Scenario: A user cannot sign into the system with an unregistered email or password
+    Given I am not signed in
+    When I navigate to the Sign In page
+    And I enter an unregistered email
+    And I enter an invalid password
+    And I click Sign In
+    Then I see an alert containing "Invalid email or password"
+    When I enter a registered email
+    And I enter an invalid password
+    And I click Sign In
+    Then I see an alert containing "Invalid email or password"
+
+  @javascript
+  Scenario: A user who is not signed in cannot access any of the user management features
+    Given I am not signed in
+    When I look at the Navigation Menu
+    Then I do not see a List Users link
+    And I do not see a Profile link
+    And I do not see a Sign Out link
+
+    When I visit the Users page directly
+    Then I am sent to the Sign In page
+    And I see an alert containing "You must be signed in to access that page"
+
+    When I visit the Add Pirate page directly
+    Then I am sent to the Sign In page
+    And I see an alert containing "You must be signed in to access that page"
+
+    When I visit the Edit Profile page directly
+    Then I am sent to the Sign In page
+    And I see an alert containing "You must be signed in to access that page"
+
+  @javascript
+  Scenario: A non-admin user cannot add a new user
+    Given I am signed in as a non-admin user
+    When I look at the Navigation Menu
+    Then I do not see an Add Pirate link
+
+    When I visit the Add Pirate page directly
+    Then I am sent to the Profile page
+    And I see an alert containing "You must be an admin user to access that page"
+
+  @javascript
+  Scenario: A non-admin user cannot delete his own profile
+    Given I am signed in as a non-admin user
+    When I navigate to the Profile page
+    And I do not see a Delete button
+    # Note: it does not seem possible to visit the "/users/:id" page with a :delete
```

Comment:
Not easily, no. Technically that's something rspec controller tests exist for, but I'm not a huge fan of those.